tests/nd_item/nd_item_async_work_group_copy_fp16.cpp

```diff
@@ -37,8 +37,10 @@ class TEST_NAME : public util::test_base {
       // Test using queue constructed already
       for_type_and_vectors<check_type, sycl::half>(queue, log,
           "sycl::half");
+#if SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS
       for_type_and_vectors<check_type, sycl::cl_half>(queue, log,
           "sycl::cl_half");
```

Comment:
The tests seem to be outdated.
Spec [says](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_data_types_2):

> For the purpose of interoperability and portability, SYCL defines a set of aliases to C++ types within the sycl::opencl namespace using the cl_ prefix. 

```suggestion
          "sycl::opencl::cl_half");
```
Right?

Reply:
I've updated most uses of `cl_*`, however there's the `cl_float4` etc vector types which also seem to be deprecated, and DPC++ doesn't alias them into `opencl::`. I've left those alone.

Reply:
> there's the `cl_float4` etc vector types which also seem to be deprecated, and DPC++ doesn't alias them into `opencl::`.

@steffenlarsen, could you please check the DPC++, please? I found https://github.com/intel/llvm/commit/e98d732406cc4ba9734cfd8ece96f6d19ddf7267 by @AlexeySachkov, which adds aliases for scalar types, but I don't see definitions for vector aliases in https://github.com/intel/llvm/blob/e89da3251187ec101f995a8dbf6e832043baebdc/sycl/include/sycl/aliases.hpp.

We need to create trackers to fix both CTS tests (i.e. `sycl::cl_float4` -> `sycl::opencl::cl_float4`) and DPC++ implementation (i.e. implement `sycl::opencl::cl_float4`).

Reply:
I found the spec defines aliases only for scalar OpenCL types https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_data_types_2.

> The OpenCL C language standard [Section 6.11](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#opencl12) defines its own built-in scalar data types, and these have additional requirements in terms of size and signedness on top of what is guaranteed by ISO C++. For the purpose of interoperability and portability, SYCL defines a set of aliases to C++ types within the sycl::opencl namespace using the cl_ prefix. These aliases are described in [Table 182](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.types.aliases).

I'm not sure what is the reason to skip vector types, but at the moment, we can open an issue for SYCL-CTS to replace OpenCL vector type aliases with sycl::vec type (i.e. `sycl::cl_float4` -> `sycl::vec<sycl::opencl::cl_float, 4>`). 

I recalled that @AlexeySachkov already filed an issue about missing vector aliases - https://github.com/KhronosGroup/SYCL-Docs/issues/335, but it's still unresolved.

Reply:
I agree with you analysis, but it sounds like DPC++ is defining the right aliases until https://github.com/KhronosGroup/SYCL-Docs/issues/335 is resolved. Please correct me if I'm wrong.